src/components/transcript/transcript.tsx

```diff
@@ -128,8 +139,10 @@ export class Transcript extends Component<TranscriptProps, TranscriptState> {
       // trigger search in case search value has changed, or new captions were added for the same language (preventSeek use-case)
       this._debounced.findSearchMatches();
     }
+  }
 
-    this._setWidgetSize();
+  componentWillMount(): void {
+    this._resizeObserver?.disconnect();
```

Comment:
Does disconnect() clean everything or is it needed to nullify the value at some point as well ?

Reply:
use redux instead of ResizeObserver